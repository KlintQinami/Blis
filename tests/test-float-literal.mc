int main()
{
  printf(1.e-2);
  printf(.1);
  printf(1.);
  printf(10.0);
  printf(1.0e3);
  printf(1e+4);
}
