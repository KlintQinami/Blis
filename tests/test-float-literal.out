0.010000
0.100000
1.000000
10.000000
1000.000000
10000.000000
